src/app/create/route.ts

```diff
@@ -0,0 +1,87 @@
+import * as Yup from "yup";
+import { db, getErrorMessage } from "../../lib";
+import { Msr, MsrPiiSec } from "../../types";
+import { Gender, MSRStatus, Race } from "@prisma/client";
+
+const payloadSchema = Yup.object({
+	zendesk_user_id: Yup.number().required(),
+	email: Yup.string().email().required(),
+	phone: Yup.string().required(),
+	first_name: Yup.string().required(),
+	last_name: Yup.string(),
+	city: Yup.string().required(),
+	state: Yup.string().required(),
+	neighborhood: Yup.string().required(),
+	zipcode: Yup.string(),
+	date_of_birth: Yup.date().nullable(),
+	color: Yup.string().oneOf(Object.values(Race)).required(),
+	gender: Yup.string().oneOf(Object.values(Gender)),
+	status: Yup.string().oneOf(Object.values(MSRStatus)).required(),
+	had_disability: Yup.boolean(),
+	accepts_online_support: Yup.boolean(),
+});
+
+export async function POST(request: Request) {
+	try {
+		if (!request.body) {
+			throw new Error("Error: Empty body!");
+		}
+		const payload = await request.json();
+
+		await payloadSchema.validate(payload);
+
+		const msr: Msr = await db.mSRs.create({
+			data: {
+				msrId: payload.zendesk_user_id,
+				gender: payload.gender ? payload.gender : "not_found",
```

Comment:
```suggestion
				gender: payload.gender
```

isso deve ser tratado la no `listener-solidarity` quando chamarmos o endpoint, nao precisa tratar aq